pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -278,6 +278,10 @@ type LocalSecretReference struct {
 const (
 	// ClusterConditionReady means the cluster is healthy and ready to accept workloads.
 	ClusterConditionReady = "Ready"
+
+	// ClusterConditionInvalidAPIEnablements means the cluster's APIEnablements are invalid.
+	// This condition usually occurs when the APIEnablements are not successfully collected.
+	ClusterConditionInvalidAPIEnablements = "InvalidAPIEnablements"
```

Comment:
Wouldn't it be better to use a more neutral descriptive word for this, such as APIEnablement completeness or something else?